package.json

```diff
@@ -69,15 +69,15 @@
         "prop-types": "^15.5.10",
         "proxyquire": "^1.7.4",
         "raf": "3.4.0",
-        "react": "16.3.2",
-        "react-dom": "16.4.2",
-        "react-wildcat-prefetch": "^7.8.0",
+        "react": "16.6.1",
+        "react-dom": "16.6.1",
+        "react-wildcat-prefetch": "8.0.0",
         "resolve": "^1.1.7",
         "rimraf": "^2.5.2",
         "shelljs": "^0.7.0",
         "sinon": "^2.0.0-pre.5",
         "sinon-chai": "^2.8.0",
-        "webpack": "^2.6.1",
+        "webpack": "3.12.0",
```

Comment:
Will going to webpack 4 break things and/or take a significant more amount of time?

Reply:
I upgraded to the same version as on web. It helped with `emotion` issues. So, I guess it is better to keep them in sync